protobuf/sync/v1/sync_service.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package sync.v1;
+
+option go_package = "flagd/grpcsync";
+
+/*
+   SyncFlagsRequest convey a key that identifies the flagd instance, enabling Sync provider to filter
+   specific flags for a specific flagd deployment.
+*/
+message SyncFlagsRequest {
+  string key = 1;
+}
+
+//SyncState convey the state of the payload
+enum SyncState {
+  // Value is ignored by the listening flagd
```

Comment:
Can you clarify this comment a bit? Perhaps mntion this is the _current_ implementation in flagd.

Reply:
Done, added detailed explanation